### PR TITLE
Fix Official Samples not loading when Gallery tab is opened

### PR DESCRIPTION
**Pull request by gitanuj**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1282)

### CODE REVIEW

File: src/GalleryViewer/GalleryViewer.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const onInit = async () => {
 
   const props: GalleryAndNotebookViewerComponentProps = {
     junoClient: new JunoClient(),
-    selectedTab: galleryViewerProps.selectedTab || GalleryTab.PublicGallery,
+    selectedTab: galleryViewerProps.selectedTab || GalleryTab.OfficialSamples,
```
Comment (srinathnarayanan) [inline src/GalleryViewer/GalleryViewer.tsx:28]: Is this change needed cos public gallery is hidden currently? since public gallery is behind a flight, should we check for the same flight here to choose between public gallery/official samples?

Comment (gitanuj) [reply src/GalleryViewer/GalleryViewer.tsx:28]: Good point. I'll add a feature flag check

**Review by srinathnarayanan — APPROVED** [review]

**Review by victor-meng — APPROVED** [review]
